src/Uno.UI/UI/Xaml/FontFamilyHelper.wasm.cs

```diff
@@ -6,19 +6,19 @@
 
 namespace Windows.UI.Xaml;
 
-public partial class FontFamilyHelper
+internal partial class FontFamilyHelper
```

Comment:
This must stay public, as it is a documented helper.

Reply:
@jeromelaban The documented helper is:

https://github.com/unoplatform/uno/blob/master/src/Uno.UI.Runtime.WebAssembly/Xaml/Media/FontFamilyHelper.cs